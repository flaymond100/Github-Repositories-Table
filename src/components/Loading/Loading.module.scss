.circularWrapper {
  text-align: center;
  margin-top: 20%;

  h3 {
    border: 0;
  }
}
