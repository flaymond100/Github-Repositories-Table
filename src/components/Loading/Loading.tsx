import { FC } from "react";
import { CircularProgress } from "@material-ui/core";
import { useTranslation } from "react-i18next";
import styles from "./Loading.module.scss";

const Loading: FC = () => {
  const [translation] = useTranslation();

  return (
    <div className={styles.circularWrapper}>
      <CircularProgress />
      <h3>
        <p>{translation("loading")}</p>
      </h3>
    </div>
  );
};

export default Loading;
