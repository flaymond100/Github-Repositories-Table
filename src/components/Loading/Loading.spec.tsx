import * as React from "react";
import { shallow } from "enzyme";
import Loading from "./Loading";

describe("render LoadingPage component", () => {
  test("should render LoadingPage component", () => {
    const component = shallow(<Loading />);

    expect(component).toMatchSnapshot();
  });
});
