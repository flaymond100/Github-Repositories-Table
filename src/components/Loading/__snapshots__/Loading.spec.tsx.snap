// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render LoadingPage component should render LoadingPage component 1`] = `
<div
  className="circularWrapper"
>
  <WithStyles(ForwardRef(CircularProgress)) />
  <h3>
    <p>
      loading
    </p>
  </h3>
</div>
`;
